codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+schema: "https://master.staging.saleor.cloud/graphql/"
+generates:
+  src/apollo/types.ts:
+    plugins:
+      - "typescript"
+    config:
+      enumsAsTypes: true
+      skipTypename: true
```

Comment:
@jwm0 should we skip typename? Dont remember why (caching in apollo?) we were using typenames in other projects.

Reply:
I would also suggest adding `dedupeOperationSuffix` to avoid generating fragment types like `UserFragmentFragment`

Reply:
done